js/testapps/evals/package.json

```diff
@@ -11,7 +11,7 @@
     "build:watch": "tsc --watch",
     "build-and-run": "pnpm build && node lib/index.js",
     "dev": "tsx --watch src/index.ts",
```

Comment:
We already have this, do we need to change genkit:dev?

Reply:
I undid this change.